samples/Client/DaprClient/Program.cs

```diff
@@ -38,11 +39,32 @@ public static async Task Main(string[] args)
             // Delete State
             await DeleteStateAsync(client);
 
+            #region Service Invoke - Required RoutingService
+            // This provides an example of how to invoke a method on another REST service that is listening on http.
+            // To use it run RoutingService in this solution.
+            // Invoke deposit operation on RoutingSample service by publishing event.
+            //await PublishDepositeEventToRoutingSampleAsync(client);
+
+            // Invoke deposit operation on RoutingSample service by POST.
+            //await InvokeWithdrawServiceOperationAsync(client);
+
+            // Invoke deposit operation on RoutingSample service by GET.
+            //await InvokeBalanceServiceOperationAsync(client);
+            #endregion
             // This provides an example of how to invoke a method on another app that is listening on http.
             // This is commented out because it requires another app to be running.
             // await InvokeMethodOnHttpServiceAsync(client);
+
```

Comment:
InvokeMethodOnHttpServiceAsync and call to it can be removed as the code which is being added covers it.

Reply:
Agree. I removed it.